jarviscli/plugins/hangman.py

```diff
@@ -1,39 +1,39 @@
 from plugin import plugin
-from random_word import RandomWords
+from random_word import RandomWords as random_words
 
 
 @plugin('hangman')
 def hangman(jarvis, s):
-    initialText = "#########################################\n" \
-                  "# Hello Hangman Game Is About To Begin! #\n" \
-                  "#     Guesses Should Be Characters!     #\n" \
-                  "#        Type 'stop' To End Game!       #\n" \
-                  "#########################################\n"
-    print(initialText)
-    randomWords = RandomWords()
-    terminateFlag = 0
+    initial_text = "#########################################\n" \
+                   "# Hello Hangman Game Is About To Begin! #\n" \
+                   "#     Guesses Should Be Characters!     #\n" \
+                   "#        Type 'stop' To End Game!       #\n" \
+                   "#########################################\n"
+    print(initial_text)
+    random_words = random_words()
```

Comment:
"""
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "/data/git/Jarvis/jarviscli/CmdInterpreter.py", line 286, in try_do
    do(self, s)
  File "/data/git/Jarvis/jarviscli/plugin.py", line 208, in run
    self._backend[0](jarvis.get_api(), s)
  File "/data/git/Jarvis/jarviscli/plugins/hangman.py", line 13, in hangman
    random_words = random_words()
UnboundLocalError: local variable 'random_words' referenced before assignment
"""

RandomWords is a class and therefore Pasal Case is okay. Other changes are good - for variables PEP8 requests in fact Snake case.